File: app/src/main/java/com/aserbao/aserbaosandroid/comon/base/viewHolder/HeadViewHolder.java
package com.aserbao.aserbaosandroid.comon.base.viewHolder;

import android.view.View;
import android.widget.TextView;

import com.aserbao.aserbaosandroid.R;
import com.aserbao.aserbaosandroid.comon.base.beans.BaseRecyclerBean;
import com.aserbao.aserbaosandroid.comon.base.interfaces.IBaseRecyclerItemClickListener;

import butterknife.BindView;
import butterknife.ButterKnife;

public class HeadViewHolder extends BaseClickViewHolder {
        @BindView(R.id.base_recycler_view_holder_head_tv)
        public TextView mBaseRecyclerViewHolderHeadTv;

        public HeadViewHolder(View itemView) {
            super(itemView);
            ButterKnife.bind(this, itemView);
        }

        public void setDataSource(BaseRecyclerBean classBean){
            String name = classBean.getName();
            mBaseRecyclerViewHolderHeadTv.setText(name);
        }

}

File: base/src/main/java/com/example/base/base/viewHolder/HeadViewHolder.kt
package com.example.base.base.viewHolder

import android.view.View
import com.example.base.base.beans.BaseRecyclerBean
import kotlinx.android.synthetic.main.base_recycler_view_holder_head.view.*

class HeadViewHolder(itemView: View?) : BaseClickViewHolder(itemView!!) {
    fun setDataSource(classBean: BaseRecyclerBean) {
        itemView.base_recycler_view_holder_head_tv.setText(classBean.name)
    }
}